guide/designing-products/user-research.md

```diff
@@ -113,7 +115,7 @@ The Bitcoin protocol works with addresses and has otherwise no sense of identity
    layout = "float-left"
 %}
 
-While we cannot know who is directly responsible for the activity, we can analyze the overall network activity. No other currency in history has had a complete and accurate database of all transactions. What can we learn from this data that allows us to improve the system for users?
+While we often can’t know who is responsible for specific single transactions, we can still analyze overall network activity. No other currency in history has offered such a complete and accurate ledger of transactions. Here’s how we can use this data to improve Bitcoin for users:
```

Comment:
> Here’s how we can use this data to improve Bitcoin for users:

This does not make sense as the following text is a new topic and not related. The original sentence was a question meant to inspire the reader to think about new ways to use the public ledger data to create better experiences.

Reply:
What do you suggest we do with it, remove it or?

Reply:
Found a good middle ground in https://github.com/BitcoinDesign/Guide/pull/365/commits/7a005dfb2b53e00b399f33ff33963a3f2a505a3d

Reply:
How about this? Makes it a little more clear that it's an appeal to the reader.

> As designers, what can we learn from this data that allows us to improve the experience of our users?

Reply:
Added in https://github.com/BitcoinDesign/Guide/pull/365/commits/66c0c6f3821190af411958971a2342857cfa0f3a not sure if that's what you meant.